.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: terraform
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+concurrency:
+  group: terraform-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Terraform fmt
+        id: fmt
+        uses: dflook/terraform-fmt@v1
+        continue-on-error: true
+      - name: Terraform Validate
+        id: validate
+        uses: dflook/terraform-validate@v1
+      - name: Terraform Plan
+        uses: dflook/terraform-plan@v1
+        id: plan
+        continue-on-error: ${{ github.event_name == 'pull_request' }}
+        with:
+          var_file: |
+            vars/tvm-ci-prod.auto.tfvars
+          workspace: tvm-ci-prod
+      - name: Terraform Apply
```

Comment:
Does this use the plan generated in plan step, or make another?

Reply:
It makes another, which isn't ideal--fixed.